integration-test/1185-garden-min-zoom.py

```diff
@@ -0,0 +1,10 @@
+# this garden previously had a min_zoom of 12, but based on its size should be
+# z16 instead.
+#
+# http://www.openstreetmap.org/way/273274870
+assert_has_feature(
+    16, 32182, 20422, 'landuse',
+    { 'kind': 'garden', 'id': 273274870, 'min_zoom': 16, 'tier': 6 })
+assert_has_feature(
```

Comment:
(My motivation here is to filter out PONIs without names server side so I can rip that out of the client stylesheet code.)